soc/ite/ec/it8xxx2/Kconfig.soc

```diff
@@ -39,6 +39,10 @@ config SOC_IT82002_AW
 	bool
 	select SOC_IT8XXX2
 
+config SOC_IT82202_BW
+	bool
+	select SOC_IT8XXX2
+
 config SOC_SERIES
 	default "it8xxx2" if SOC_SERIES_ITE_IT8XXX2
```

Comment:
Same with this, the names should match e.g. `default "it8xxx2" if SOC_SERIES_IT8XXX2` or `default "ite_it8xxx2" if SOC_SERIES_ITE_IT8XXX2`

Reply:
Remove "_ITE".